.evergreen/.evg.yml

```diff
@@ -821,6 +821,40 @@ functions:
           
           MONGODB_URI="${MONGODB_URI}" JAVA_VERSION="${JAVA_VERSION}" .evergreen/run-csfle-tests-with-mongocryptd.sh
 
+  "trace artifacts":
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        script: |
+           tag=$(git describe --tags --always --dirty)
+
+           # remove the leading 'r'
+           version=$(echo -n "$tag" | cut -c 2-)
+
+           cat <<EOT > trace-expansions.yml
+           release_version: "$version"
+           EOT
+           cat trace-expansions.yml
+    - command: expansions.update
+      params:
+        file: src/trace-expansions.yml
+    - command: papertrail.trace
+      params:
+        key_id: ${papertrail_key_id}
+        secret_key: ${papertrail_secret_key}
+        product: ${product}
+        version: ${release_version}
+        filenames:
+          - "src/build/repo/org/mongodb/*/*/*.jar"
```

Comment:
Could all these be replaced with one line, something like:

`"src/build/repo/org/mongodb/*/*/*"`

?

Reply:
Unfortunately not -- I discovered that there are multiple files (with the same name) included in these directories that document what we pushed, but are not considered release artifacts. They do not show up on in the remote repos, and are some kind of metadata xml file. I agree it'd be nicer to read if we could, though